app/build.gradle.kts

```diff
@@ -28,10 +28,6 @@ android {
 
     buildTypes {
         debug {
-            isDebuggable = true  // Enable debugging features
```

Comment:
i was referring only to the 

> buildConfigField("String", "BASE_URL", "\"https://dev.example.com/api\"")

i think the removal of other lines will cause problems with Firebase.